sdks/python/apache_beam/options/pipeline_options_test.py

```diff
@@ -192,47 +192,53 @@ def _add_argparse_args(cls, parser):
     options = PipelineOptions(['--redefined_flag'])
     self.assertTrue(options.get_all_options()['redefined_flag'])
 
+  # TODO(mariagh): Require unique names only within a test.
```

Comment:
It will be hard to keep this system, especially across files. Perhaps you can prefix options with test name (or file name) to have unique names.